### PR TITLE
added project for web_portal

### PR DESCRIPTION
**Pull request by BetwinX**

### CODE REVIEW

**Comment by vincalw78** [comment]
Add  home and about pages endpoints

**Review by vincalw78 — COMMENTED** [review]
Add home and about pages endpoints

File: web_portal/server.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello, World!'
```
Comment (vincalw78) [inline web_portal/server.py:6]: Add home and about pages endpoints

Comment (BetwinX) [reply web_portal/server.py:6]: Will be added in the next tasks in the other branches

**Review by vincalw78 — APPROVED** [review]

**Pull request merged**